dotnet/samples/KernelSyntaxExamples/Example09_FunctionTypes.cs

```diff
@@ -39,35 +39,26 @@ public static async Task RunAsync()
         await kernel.InvokeAsync(plugin["type03"]);
         await kernel.InvokeAsync(kernel.Plugins["test"]["type03"]);
 
-        await kernel.InvokeAsync(plugin["type04"], variables);
-        await kernel.InvokeAsync(kernel.Plugins["test"]["type04"], variables);
+        await kernel.InvokeAsync(plugin["type04"]);
+        await kernel.InvokeAsync(kernel.Plugins["test"]["type04"]);
 
-        await kernel.InvokeAsync(plugin["type05"], variables);
-        await kernel.InvokeAsync(kernel.Plugins["test"]["type05"], variables);
+        await kernel.InvokeAsync(plugin["type05"], arguments);
```

Comment:
IF there are no arguments we shouldn't need to pass this, same for others.

This might be here due to the old `SKContext`.

```suggestion
        await kernel.InvokeAsync(plugin["type05"]);
```